internal/driver/constant.go

```diff
@@ -36,6 +36,8 @@ const (
 	// RAW_TYPE define binary data type which read from Modbus device
 	RAW_TYPE = "rawType"
 
+	// WORD_LENGTH  E.g. "abcd" need 4 bytes as is 2 word, so WORD_LENGTH=2
+	WORD_LENGTH            = "wordLength"
```

Comment:
Rename the `wordLength` to `stringRegisterSize` and check the register size should be within the range of 1~123.

The register size limit can refer to https://modbus.org/docs/Modbus_Application_Protocol_V1_1b3.pdf
> 6.12 16 (0x10) Write Multiple registers
This function code is used to write a block of contiguous registers (1 to 123 registers) in a
remote device.